### PR TITLE
Added Microsoft Translator

### PR DESCRIPTION
**Pull request by luixxiul**

### CODE REVIEW

File: src/chrome/content/rules/Microsoft_Translator.com.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<ruleset name="Microsoft Translator">
+
+	<target host="api.microsofttranslator.com" />
+	<target host="docs.microsofttranslator.com" />
+	<target host="hub.microsofttranslator.com" />
+	<target host="labs.microsofttranslator.com" />
+	<target host="ssl.microsofttranslator.com" />
+	<target host="www.microsofttranslator.com" />
+
+	<exclusion pattern="^http://labs\.microsofttranslator\.com/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/Microsoft_Translator.com.xml]: Please document why this is needed.

**Comment by J0WI** [comment]
You should also check:
translatoruser.net
ssl.translatoruser.net
www.translatoruser.net

File: src/chrome/content/rules/Microsoft_Translator.com.xml (hunk 1 above)
Comment (luixxiul) [inline src/chrome/content/rules/Microsoft_Translator.com.xml]: My mistake. I am not sure why I added this here

**Comment by luixxiul** [comment]
@J0WI translatoruser.net -> DNS_PROBE_FINISHED_NXDOMAIN

**Comment by luixxiul** [comment]
@J0WI Updated.

**Comment by J0WI** [comment]
Therefore Translatoruser.net.xml should also be linked with Microsoft.xml.

**Comment by luixxiul** [comment]
@J0WI OK, linked them with each other.

**Comment by J0WI** [comment]
LGTM, thanks!

**Pull request merged**